crates/server/src/admin/handlers/context.rs

```diff
@@ -244,8 +244,8 @@ pub async fn get_context_storage_handler(
 
 #[derive(Deserialize)]
 pub struct JoinContextRequest {
-    pub public_key: PublicKey,
-    pub private_key: [u8; 32],
+    pub public_key: Option<PublicKey>,
+    pub private_key: Option<[u8; 32]>,
```

Comment:
I don't understand this change. I think you're right to change something here, but this isn't the change to make.

We only need `Option<PrivateKey>`, and nothing else.

Reply:
public key removed as we can generate it from private key